source/server/admin/stats_handler.cc

```diff
@@ -132,13 +134,23 @@ Http::Code StatsHandler::handlerPrometheusStats(absl::string_view path_and_query
   const Http::Utility::QueryParams params =
       Http::Utility::parseAndDecodeQueryString(path_and_query);
   const bool used_only = params.find("usedonly") != params.end();
+
+  const absl::optional<std::string> export_text_readouts_value =
```

Comment:
It would be preferable if this used the same true/false logic as `usedonly` above. I don't really like the `usedonly` logic very much, but I'd rather have it be consistent. 

I think the name can be `text_readouts` (slightly less verbose than `export_text_readouts`)

Reply:
Done